app/views/home/_topannounce.html.erb.template

```diff
@@ -0,0 +1,16 @@
+<%#
+  Use this file as an opportunity to broadcast a permanent message on
+  the root '/' path at the bottom of the page. This is normally the page
+  with a list of all courses.
+
+  For example, at CMU we use it to provide students and instructors with
+  a convenient link to an older deployment of Autolab that exists solely
+  for archival purposes.
+%>
```

Comment:
I think this explanation should also say that you need to rename the file for this to work.  Also, I don't know if our older courses html should be included in this file if we're trying to remove CMU-specific stuff.  If you want to leave it as an example, maybe leave a comment to explain that it's an example/comment the code out.